ci/test_common.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: BSD-3-Clause
+
+set -euo pipefail
+
+source "$(dirname "$0")/test_utils.sh"
```

Comment:
Can we just pull test_utils.sh into this file, or just put all this new functionality directly into test_utils.sh? AFAICT there's no reason for them to be separate and we always need both.

Reply:
Done in https://github.com/rapidsai/ucxx/pull/167/commits/9e6ce880d5101076be031161f1fbeed3a4337e33